test/end-to-end.js

```diff
@@ -1185,3 +1204,9 @@ function readBuiltinPhasesFromSanbox() {
   app.lazyrouter(); // initialize request handling phases
   return app._requestHandlingPhases;
 }
+
+function getSandBoxApp() {
+  var app = require(SANDBOX);
+  app.use(app.get('restApiRoot'), app.loopback.rest());
```

Comment:
No no, the REST adapter must be added by loopback-boot. If it isn't, then we should skip the test for now.

Please remove this line.

```js
function loadSandboxApp() {
  var app = require(SANDBOX);
  // TODO wait until the app boots
  return app;
}
```